### PR TITLE
[ci] add xsbench integration workflow

### PR DESCRIPTION
**Pull request by perarnau**
No spack, direct checkout of xsbench master branch.

### CODE REVIEW

**Comment by perarnau** [comment]
Currently fails, which might indicate an actual issue.

**Review by NicolasDenoyelle — DISMISSED** [review]
All tests passed.
I added a check on the application result checksum to make sure the application ran and
gave its expected result.

**Review by NicolasDenoyelle — DISMISSED** [review]
Some extra merging flew under my radar. This is resulted in changing the formatting but it is not a problem.

File: include/Makefile.am [hunk 1]
```diff
@@ -4,6 +4,7 @@ include_aml_areadir=$(includedir)/aml/area
 include_aml_area_HEADERS = aml/area/linux.h
 
 include_aml_higherdir=$(includedir)/aml/higher
+
```
Comment (perarnau) [inline include/Makefile.am:7]:
```suggestion

```

File: include/Makefile.am [hunk 2]
```diff
@@ -42,7 +43,7 @@ include_amlutils_HEADERS = \
 
 if HAVE_HWLOC
 include_aml_area_HEADERS+= aml/area/hwloc.h
-include_aml_replicaset_HEADERS += aml/higher/replicaset/hwloc.h
+include_aml_replicaset_HEADERS+=aml/higher/replicaset/hwloc.h
```
Comment (perarnau) [inline include/Makefile.am:46]:
```suggestion
include_aml_replicaset_HEADERS += aml/higher/replicaset/hwloc.h
```

**Review by NicolasDenoyelle — APPROVED** [review]

**Pull request merged**